src/backend/constants.ts

```diff
@@ -28,7 +28,7 @@ const fontsStore = new TypeCheckedStoreBackend('fontsStore', {
 })
 
 const isMac = process.platform === 'darwin'
-const isIntelMac = isMac && cpus()[0].model.includes('Intel') // so we can have different behavior for Intel Mac
+const isIntelMac: boolean = isMac && cpus()[0].model.includes('Intel') // so we can have different behavior for Intel Mac
```

Comment:
weird, shouldn't this be inferred as boolean? `isMac` is boolean and `includes` returns a boolean